client/ayon_xstudio/addon.py

```diff
@@ -0,0 +1,53 @@
+"""Declare the xStudio Addon."""
+
+from __future__ import annotations
+
+import os
+from typing import Dict, List, Optional
```

Comment:
There is `annotations` import so you can use `list` and `dict`.
```suggestion
from typing import Optional
```